### ISSUE TITLE
Multiple use tags per phase

### ISSUE THREAD

**Issue by jdub** [hashicorp/waypoint#688]
I was curious about more complex builds and deployments, hoping there'd be a way to have multiple use tags in a phase. @evanphx mentioned it was [under investigation](https://discuss.hashicorp.com/t/waypoint-deploy-to-multiple-locations/16441/2?u=jdub) on the forum, so I thought I'd make a note of it over here. 😅

**Comment by mitchellh** [comment]
Yes this is planned for the future. This won't be in 0.2 but perhaps 0.3. It is something we really want. :)
